app/views/admin/admin_invitations/index.html.erb

```diff
@@ -10,52 +10,63 @@
+    <p>
+      <%= t(".send_to_email.description") %>
+      <%= text_field_tag "invitation[invitee_email]",
+            (@invitation.try(:invitee_email) || ""),
+            title: t(".send_to_email.invite_by_email_title") %>
+      <span class="submit actions"><%= submit_tag t(".send_to_email.invite_user") %></span>
     </p>
   </fieldset>
 <% end %>
 
-<%= form_tag url_for(:controller => 'admin/admin_invitations', :action => 'invite_from_queue') do %>
+<%= form_tag url_for(controller: "admin/admin_invitations", action: :invite_from_queue), class: "queue invitation simple post", autocomplete: "off" do %>
   <fieldset class="simple">
-    <h3 class="heading">Send invite codes to people in our <%= link_to 'invitations queue', invite_requests_path %></h3>
-    <p>There are <%= InviteRequest.count %> requests in the queue.</p>
+    <h3 class="heading">
+      <%= t(".invite_from_queue.heading_html",
+            invitations_queue_link: link_to(t(".invite_from_queue.invitations_queue"), invite_requests_path)) %>
+    </h3>
+    <p><%= t(".invite_from_queue.requests_in_queue", count: InviteRequest.count) %></p>
     <p>
-      <%= label_tag "invitation[invite_from_queue]", ts('Number of people to invite') %>: <%= text_field_tag "invitation[invite_from_queue]" %>
-      <span class="submit actions"><%= submit_tag ts('Invite from queue')  %></span>
+      <%= label_tag "invitation[invite_from_queue]", t(".invite_from_queue.number_to_invite") %>
+      <%= text_field_tag "invitation[invite_from_queue]" %>
+      <span class="submit actions"><%= submit_tag t(".invite_from_queue.invite_from_queue") %></span>
     </p>
   </fieldset>
 <% end %>
 
-<%= form_tag  url_for(:controller => 'admin/admin_invitations', :action => 'grant_invites_to_users') do  %>
+<%= form_tag url_for(controller: "admin/admin_invitations", action: :grant_invites_to_users), class: "bulk invitation simple post", autocomplete: "off" do %>
   <fieldset>
-    <h3 class="heading"><%=h 'Give invite codes to current users' %></h3>
+    <h3 class="heading"><%= t(".grant_invites.heading") %></h3>
     <dl>
-      <dt><%= label_tag "invitation[number_of_invites]", ts('Number of invitations') %>:</dt>
+      <dt><%= label_tag "invitation[number_of_invites]", t(".grant_invites.number_of_invitations") %></dt>
       <dd><%= text_field_tag "invitation[number_of_invites]" %></dd>
-      <dt><%= label_tag "invitation[user_group]", ts('Users') %>:</dt>
-      <dd><%= select_tag "invitation[user_group]", "<option>All</option><option>With no unused invitations</option>".html_safe  %></dd>
-      <dt class="landmark">Submit</dt>
-      <dd class="submit actions"><%= submit_tag "Generate invitations" %></dd>
+      <dt><%= label_tag "invitation[user_group]", t(".grant_invites.users") %></dt>
+      <dd>
+        <%= select_tag "invitation[user_group]",
+              options_for_select([t(".grant_invites.all"), t(".grant_invites.with_no_unused")],
+                t(".grant_invites.all")) %>
```

Comment:
From [the docs](https://api.rubyonrails.org/classes/ActionView/Helpers/FormOptionsHelper.html#method-i-options_for_select), so this doesn't break on us if we ever translate it: 

> Given a container where the elements respond to first and last (such as a two-element array), the “lasts” serve as option values and the “firsts” as option text.

```suggestion
              options_for_select([[t(".grant_invites.all"), "All"], [t(".grant_invites.with_no_unused")], "With no unused invitations"], "All") %>
```